launcher/teeserver/tee_server.go

```diff
@@ -148,12 +148,6 @@ func (a *attestHandler) attest(w http.ResponseWriter, r *http.Request, client ve
 			return
 		}
 
-		if tokenOptions.Audience == "" {
```

Comment:
@JoshuaKrstic Does this align with our expected usage/behavior for the non-default IPC endpoint? My instinct is this PR should focus on the default case.

Reply:
Hm actually, now that I think about it, we should just have the GET and the auto-refresh pass their own token options to the agent instead of filling in the audience in the agent.

Reply:
After thinking about it more, you made the right call overriding it in the agent. Just keep this condition and we should be good to go.